script/setup

```diff
@@ -19,11 +19,17 @@ NO_COLOR='\033[0m'
 
 # Check ruby version
 RUBY_VERSION=$(cat .ruby-version)
-if command -v rbenv > /dev/null; then
+if command -v rbenv > /dev/null && ! command rvm > /dev/null; then
```

Comment:
` && ! command rvm > /dev/null` is this really needed ? Presumably if you have `rbenv` installed, you shouldn't have `rvm` install. If I remember correctly they don't work nicely together.

Reply:
I thought so to, so I've added a suggested commit to demonstrate.